### PR TITLE
kub: converted to single file format

### PR DESCRIPTION
**Pull request by FFHener**

### CODE REVIEW

File: locations/kub.yml [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+
+snmp_devices:
+  - hostname: kub-simeon
+    address: 10.31.139.2
+    snmp_profile: airos_8
+
+
 ipv6_prefix: "2001:bf7:830:8800::/56"
 
 # Rio-Reiser got following prefixes:
```
Comment (Noki) [inline locations/kub.yml:28]: this looks like an artefact from copying a config. might be good to change this in the process.

**Comment by pktpls** [comment]
LGTM :+1: I'll remove that stray comment separately

**Comment by pktpls** [comment]
> I'll remove that stray comment separately

Uff, branch protection, I love it...

File: locations/kub.yml (hunk 1 above)
Comment (FFHener) [reply locations/kub.yml:28]: I made a Issue #725

**Pull request merged**